css/styles.css

```diff
@@ -0,0 +1,301 @@
+*{
+    --bg-color: rgb(248, 244, 245);
+    box-sizing: border-box;
+}
+body {
+    font-size: 16px;
+    color: var(--bg-color);
+    font-family: lato;
+    margin:0;
+    letter-spacing: 1px;    
+}
+h1 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 50px;
+    background-color: #6633cc;
+    margin: 6px;
+    color: #fff;
+    padding: 0px 30px 0px 20px;
+    display: table-caption;
+}
+h2 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 40px;
+    letter-spacing: 1px;
+    color: rgb(16, 169, 237);
+}
+h4 {
+    margin: 0;
+}
+.navTop {
+    position: fixed;
+    background-image: url("../images/header_bg.png");
+    width: 100%;
+    height: 70px;
+    display: table-caption;
+}
+
+.left {
+    display: inline-block;
+    padding-left: 60px;
+    position: absolute;
+}
+.left img {
+    width: 100px;
+    margin-top: 4px;
+} 
+.right {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    float: right;
+}
+.right li {
+    display: inline;
+}
+.right li a{
+    padding: 10px 20px 0px 0px;
+    color: white;
+    font-size: 14px;
+    text-decoration: none;
+    display: inline-block;
+    margin:17px;
+}
+.right a:hover {
+    text-decoration: underline;
+}
+.right button {    
+    border: 2px solid #fff;
+    border-radius: 20px;
+    padding: 10px 20px;
+    font-size: 14px;
+    margin-right: 30px;
+    font-weight: 600;
+    color: #fff;
+    letter-spacing: 1px;
+    text-decoration: none;
+    background-color: transparent;
+    box-sizing: border-box;
+}
+.segment-one button {
```

Comment:
Is better to assign class names related to the content instead the position or the styling